packages/legend-application-query/src/components/QueryEditorComponentTestUtils.tsx

```diff
@@ -198,3 +205,64 @@ export const TEST__setUpQueryEditor = async (
   );
   return renderResult;
 };
+
+export const TEST_setUpQueryBuilderState = async (
```

Comment:
Nice, but let's move this to `QueryBuilderTestUtils` since this is purely a state testing tool